packages/main/src/Card.hbs

```diff
@@ -1,42 +1,12 @@
 <div
-	class="{{classes.main}}"
+	class="{{classes}}"
 	dir="{{effectiveDir}}"
 	role="region"
 	aria-label="{{ariaLabelText}}"
 	aria-labelledby="{{ariaLabelledByCard}}">
 	{{#if hasHeader}}
```

Comment:
this conditional is no longer needed

Reply:
I think we need it. How else will it be decided  if we need ui5-card--nocontent?

Reply:
No, I meant the `{{#if hasHeader}}` on line 7.